client/landing/jetpack-cloud/components/scan-threats/style.scss

```diff
@@ -11,7 +11,7 @@
 		margin-top: 24px;
 	}
 
-	&__run-scan-button {
+	&__run-scan-button.scan-threats__run-scan-button {
```

Comment:
Just out of curiosity, and I haven't tested because I can't access this page, but would adding an `is-link` class prevent requiring some of the overrides here? https://github.com/Automattic/wp-calypso/blob/62ea5ba7e06f88ba8060ba6516dee2ae3741914b/client/landing/jetpack-cloud/components/scan-threats/index.tsx#L58